am2320/am2320.go

```diff
@@ -0,0 +1,181 @@
+// Copyright 2024 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+//
+// This package provides a driver for the AOSONG AM2320 Temperature/Humidity
+// Sensor. This sensor is a basic, inexpensive i2c sensor with reasonably good
+// accuracy for both temperature and humidity.
+//
+// The datasheet is located at:
```

Comment:
```
// Datasheet
```

will create a title. It is documented at https://go.dev/doc/comment#headings

You can see it locally with:

```
go install golang.org/x/pkgsite/cmd/pkgsite@latest
pkgsite -open .
```